docs/tutorials/kiss-icp.md

```diff
@@ -0,0 +1,92 @@
+# Kiss-icp
+
+## Install kiss-icp pipeline
+
+
+```sh
+pip install kiss-icp
+```
+
+Next to test if the system is working are to understand instructions on how to run it do the following command
+
+```sh
+kiss_icp_pipeline --help
+```
+
+# Install kiss-icp ROS wrappers
+
+From the src folder:
+
+```sh
+git clone https://github.com/PRBonn/kiss-icp
+```
+
+Now go to the wd and build it by running the following command:
+
+```sh
+colcon build
+```
+
+If you are having problems with other packages, you can try to only build the kiss-icp
+
+```sh
+colcon build --packages-select kiss_icp
+```
+
+Finally run the following command to finish the build
+
+```sh
+source ./install/setup.bash
+```
+
+# How to run
+
+The only required argument to provide is the topic name so KISS-ICP knows which PointCloud2 to process:
+
+```sh
+ros2 launch kiss_icp odometry.launch.py bagfile:=<path_to_rosbag> topic:=<topic_name>
+```
+
+You can optionally launch the node with any bagfile, and play the bagfiles on a different shell:
+```sh
+ros2 launch kiss_icp odometry.launch.py topic:=<topic_name>
+```
+and then,
+
+```sh
+ros2 bag play <path>*.bag
+```
+
+The topic_name should be the topic of the bagfile. In case you do not know, run the following commands and see which topic has the lidar data from the bagfile, and then repeat the commands above:
+
+```sh
+ros2 bag play <path>*.bag
+ros2 topic list
+```
+
+
+
+
+To get the pose from kiss_icp, in a new terminal you should run:
+
+```sh
+ros2 topic echo /kiss/odometry
+```
+
+# Internal structure of the package
+
+There are to nodes, one for kiss-icp and another one for rviz2, to visualize the simulation. 
```

Comment:
There is only one node that is programmed inside the package, this refers to the nodes being launched, rviz is not a node from kiss-icp package